Library/Formula/ngircd.rb

```diff
@@ -36,6 +36,37 @@ def install
     args << "--with-pam" if build.with? "pam"
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
+  end
+
+  plist_options :manual => "ngircd"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_sbin}/ngircd</string>
+          <string>--nodaemon</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>KeepAlive</key>
+        <true/>
+        <key>StandardErrorPath</key>
+        <string>#{var}/Logs/ngIRCd.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/Logs/ngIRCd.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
```

Comment:
Is this `plist` included with the software itself? If so, can we install it itself?